dask/base.py

```diff
@@ -356,9 +361,17 @@ def visualize(*args, **kwargs):
     optimize_graph : bool, optional
         If True, the graph is optimized before rendering.  Otherwise,
         the graph is displayed as is. Default is False.
+    color: False
```

Comment:
Perhaps `color : {None, 'order'}, optional` instead of `color: False` (see parameters section in https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections).

`None` would take the place of `False` then.

Reply:
Also, should add the `color` kwarg to the method docstring above.

Reply:
Thanks.  Changed.